tox.ini

```diff
@@ -17,7 +17,7 @@ deps =
 [testenv:scripttests]
 commands = nosetests {toxinidir}/script-tests []
 setenv = 
-  PYTHONPATH = {toxinidir}/perfkitbenchmarker/scripts:{env:PYTHONPATH}
+  PYTHONPATH = {toxinidir}/perfkitbenchmarker/scripts:{envdir}
```

Comment:
I can't commit without this one, since pre-commit tests will fail. This issue (#900) has been around for some time.